Source/Windows/ARM64EC/Module.cpp

```diff
@@ -291,7 +291,13 @@ static void LoadStateFromECContext(FEXCore::Core::InternalThreadState* Thread, C
 
   if ((Context.ContextFlags & CONTEXT_FLOATING_POINT) == CONTEXT_FLOATING_POINT) {
     // Floating-point register state
-    CTX->SetXMMRegistersFromState(Thread, reinterpret_cast<const __uint128_t*>(Context.FltSave.XmmRegisters), nullptr);
+    if ((Context.ContextFlags & CONTEXT_XSTATE) == CONTEXT_XSTATE) {
```

Comment:
Is this equality operator expecting `CONTEXT_XSTATE` or is it meant to be checking for `CONTEXT_ARM64_FEX_YMMSTATE` which is set during the store state side of things?

Reply:
The context here is an x64 context which stores avx state as part of xstate (this is the same as on regular x64 wine), the CONTEXT_ARM64_FEX_YMMSTATE flag is for indicating that an arm64ec context packed into an arm64 context contains avx state in the higher 16 vector registers. See the corresponding wine patch for the translation between the two

Reply:
Ah, okay I see what you mean after looking closelier at the wine patch.